README.md

```diff
@@ -21,7 +21,7 @@ Add the plugin to your npm-project:
 $ npm install semantic-release-slack-bot -D
 ```
 
-The corresponding slack app has be installed in you slack workspace as well. Follow the instructions under [configuration](#configuration) for more information.
+The corresponding slack app has to be installed in your slack workspace as well. Follow the instructions under [configuration](#configuration) for more information.
```

Comment:
Great that you have cleaned up in the Readme! I didn't know that there were that many spelling mistakes 😅